README.md

```diff
@@ -918,6 +918,23 @@ Options for the [OpenId Connect](http://developer.okta.com/docs/api/resources/oi
       authScheme: 'OAUTH2'
     }
     ```
+- **customButtons** - Adds an array of custom Auth buttons configurable by the properties below:
+
+    - `title` - String to set button text
+    - `className` - Sets a class on the button
+    - `dataAttr` - Sets data-se attribute on the button
+    - `click` - Callback function that is executed when button is clicked
+
+    ```javascript
+    customButtons: [{
+      title: 'Click Me',
+      className: 'btn-customAuth',
+      dataAttr: 'custom-data',
+      click: function() {
+        //custom callback
```

Comment:
nit: space between // and custom 

I do think it's more useful to add a real example, i.e. you could show the code to redirect to a different url (to give more context on what they'll actually be doing. With a comment before it saying that this is an example showing a redirect to bla.